dist/Dockerfile.e2e-ubi8/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN dnf update -y \
 RUN bash -c "source /opt/app-root/etc/scl_enable && hack/build_e2e.sh"
 
 FROM registry.ci.openshift.org/openshift/release:golang-1.13 AS golang_builder
-RUN env GOBIN=/usr/local/bin go get -u github.com/tsenart/vegeta
+RUN curl -L https://github.com/tsenart/vegeta/releases/download/v12.8.4/vegeta_12.8.4_linux_amd64.tar.gz| tar xvzf - -C /usr/local/bin/ vegeta
```

Comment:
[Earlier compilation error][1] was:

```
STEP 11: RUN env GOBIN=/usr/local/bin go get -u github.com/tsenart/vegeta
# golang.org/x/net/http2
/go/src/golang.org/x/net/http2/transport.go:598:14: undefined: tls.Dialer
```

Is that just because of our ancient golang-1.13 from the line above?

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cincinnati/422/pull-ci-openshift-cincinnati-master-stablerust-e2e/1389575368049954816/build-log.txt

Reply:
Also https://github.com/tsenart/vegeta/releases/tag/v12.8.4 is released before approximate 7 months of so. Wondering why the latest version can not be downloaded with `go get -u github.com/tsenart/vegeta`

Reply:
Seems its not pinned to a particular `golang.org/x/net/http2`, so at some point it was updated (or some other dependency pulled it in?) and requires Go 1.17 now.

In any case, we don't need to be rebuilding vegeta every e2e run, its faster and more reliable to use prebuilt binaries

Reply:
I will check if this will impact the downstream brew builds.